pkg/subsystem/entity/entities.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package entity
```

Comment:
A single small file is generally not the right granularity for Go packages:
https://dave.cheney.net/practical-go/presentations/qcon-china.html#_consider_fewer_larger_packages

I would move this file into subsystem package.
Also I would move match/match.go into subsystem package, since it works only with Subsystem object is very tightly coupled to it.
Then everything else in the match package becomes helpers for the linux package and can be moved there and unexported.
This way we can get rid of 2 package.